packages/typescript/src/index.ts

```diff
@@ -42,11 +46,40 @@ export default function typescript(options: RollupTypescriptOptions = {}): Plugi
           parsedOptions,
           writeFile(fileName, data) {
             emittedFiles.set(fileName, data);
+          },
+          status(diagnostic) {
+            // Fullfil watcher promises by Typescript diagnostic message codes
+            if (diagnostic.category === ts.DiagnosticCategory.Message) {
+              switch(diagnostic.code) {
+                case 6032:
```

Comment:
Constants should be used for these codes.

Reply:
Typescript didn't include constants for the codes. I will add the relevant.